src/features/ThreadHistory/ThreadHistory.module.css

```diff
@@ -0,0 +1,10 @@
+.calloutContainer {
+  position: absolute;
+  left: 50%;
+  top: 30px;
+  transform: translateX(-50%);
+  width: fit-content;
+  z-index: 1;
+  display: flex;
+  justify-content: center;
+}
```

Comment:
is the pre selector used here?

Reply:
No, pre selector is used in a code block area, `calloutContainer` is used only for info or error bubbles